features/support/env.rb

```diff
@@ -4,11 +4,31 @@
 module ArubaExt
   def run_command(cmd, timeout = nil)
     exec_cmd = cmd =~ /^rspec/ ? "bin/#{cmd}" : cmd
-    # Ensure bundler env vars are unset
     unset_bundler_env_vars
-    # Ensure the correct Gemfile is found
+    # Ensure the correct Gemfile and binstubs are found
     in_current_directory do
-      super(exec_cmd, timeout)
+      with_unbundled_env do
```

Comment:
I'll chip in to say that this call to `with_unbundled_env` is probably not needed.

Reply:
So I tried [without](https://github.com/rspec/rspec-rails/commit/3a1f696f1241d48973f49d4c2147bf2493453f4f)

And [it breaked](https://travis-ci.org/rspec/rspec-rails/builds/632604861) the build. So I rollback.

Reply:
That's surprising and I'm a little concerned that I don't know what is going on in my own fix, but if it works for now, I guess it's ok. Thanks for checking this, anyway.